compliance/admin.py

```diff
@@ -0,0 +1,31 @@
+"""
+Admin site bindings for compliance
+"""
+
+from django.contrib import admin
+from users.utils import ensure_active_user
+
+from .constants import RESULT_MANUALLY_APPROVED
+from .models import ExportsInquiryLog
+
+
+class ExportsInquiryLogAdmin(admin.ModelAdmin):
+    """Admin for ExportsInquiryLog"""
+
+    model = ExportsInquiryLog
+    search_fields = ["user__email", "computed_result", "info_code", "reason_code"]
+    list_filter = ["computed_result", "info_code", "reason_code"]
+    list_display = ["user", "computed_result", "info_code", "reason_code"]
+
+    def save_model(self, request, obj, form, change):
```

Comment:
I think we need to do this differently, I think having the entire model directly editable in the admin makes this error-prone. Can you rework this PR to do the following?

- Make all fields read-only so they can't accidentally be changed
- Can you implement the manual approval as a django-admin action instead that changes `computed_result` to `RESULT_MANUALLY_APPROVED`? https://docs.djangoproject.com/en/2.2/ref/contrib/admin/actions/
- Only update the record(s) if the user's `computed_result` is not `RESULT_SUCCESS` or `RESULT_MANUALLY_APPROVED`

Reply:
Done.